src/components/v1/V1Project/Pay/V1PayButton.tsx

```diff
@@ -0,0 +1,137 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Tooltip } from 'antd'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { useContext, useState } from 'react'
+import { formatWad, fromWad } from 'utils/formatNumber'
+import { decodeFundingCycleMetadata } from 'utils/v1/fundingCycle'
+import AmountToWei from 'utils/AmountToWei'
+import { usePayV1ProjectTx } from 'hooks/v1/transactor/PayV1ProjectTx'
+
+import PayWarningModal from 'components/shared/PayWarningModal'
+
+import { readNetwork } from 'constants/networks'
+import { disablePayOverrides } from 'constants/v1/overrides'
+import { V1_PROJECT_IDS } from 'constants/v1/projectIds'
+import { V1_CURRENCY_ETH, V1_CURRENCY_USD } from 'constants/v1/currency'
```

Comment:
**question**: couldn't this also use the new currencyContext now?

Reply:
I was thinking the currencyContext is only necessary for version-agnostic components, and that it might be simpler (at least for this PR) to continue using `V1_CURRENCY...` for version-specific components

Reply:
Yeah sweet that makes sense 👍